src/deepsparse/evaluation/cli.py

```diff
@@ -210,7 +210,7 @@ def main(
         **integration_args,
     )
 
-    _LOGGER.info(f"Evaluation done. Results:\n{result}")
+    _LOGGER.info(f"Evaluation done. Result:\n{result.formatted}")
```

Comment:
For now, we just print `result.formatted`. In the next PR, I will change it, so we print a pretty table to the user.